jwst/datamodels/schemas/ifucube.schema.yaml

```diff
@@ -18,6 +18,10 @@ allOf:
           title: EXTNAME of the extension containing data quality mask
           type: string
           fits_keyword: MASKEXT
+        weightmap_extension:
+          title: EXTNAME of the extension containing weight map
+          type: string
+          fits_keyword: WMAPEXT
```

Comment:
Did someone ask to have WMAPEXT added as another keyword? The FLUXEXT, ERREXT, and MASKEXT were asked for by David Law a couple years ago, because that's what some other observatories are using in their IFU data, and hence we want ours to be consistent so that data analysis tools can find things. Is anyone else using WMAPEXT?